Rendera/include/camera.hpp

```diff
@@ -15,6 +16,9 @@ class Camera{
     int img_height;
     float focal_length;
     //viewport screen at (x = focal_length) (viewing direction is +X)
+
+    //random number generator member
+    std::minstd_rand random;
```

Comment:
Since having it as a static member variable is giving us linker errors, why don't we make a private member function called `random_int` or something, which will return a random integer, and move the declaration of this static variable inside that function?